src/main/java/decodes/tsdb/algo/BigAdder.java

```diff
@@ -1,123 +1,154 @@
+import org.opendcs.annotations.algorithm.Output;
 
-//AW:JAVADOC
 /**
-Takes up to 20 input values labeled input1 ... input20. Multiplies
-them by coefficients supplied in properties coeff1 ... coeff20.
-Adds them together and produces a single output labeled 'output'.
-The property input_constant is added to the ouput result.
-input_constant defaults to 0 if not supplied
-Values not assigned by computation are ignored.
-All coefficients default to 1.0 if not supplied.
- 
 This algorithm is a modification of Scaler adder
 Created by M. Bogner   May 2009
  */
-//AW:JAVADOC_END
+@Algorithm(
+		description ="Takes up to 20 input values labeled input1 ... input20. Multiplies\n" +
+				"them by coefficients supplied in properties coeff1 ... coeff20.\n" +
+				"Adds them together and produces a single output labeled 'output'.\n" +
+				"The property input_constant is added to the ouput result.\n" +
+				"input_constant defaults to 0 if not supplied\n" +
+				"Values not assigned by computation are ignored.\n" +
+				"All coefficients default to 1.0 if not supplied." )
 public class BigAdder extends decodes.tsdb.algo.AW_AlgorithmBase
 {
-//AW:INPUTS
-	public double input1;	//AW:TYPECODE=i
-	public double input2;	//AW:TYPECODE=i
-	public double input3;	//AW:TYPECODE=i
-	public double input4;	//AW:TYPECODE=i
-	public double input5;	//AW:TYPECODE=i
-	public double input6;	//AW:TYPECODE=i
-	public double input7;	//AW:TYPECODE=i
-	public double input8;	//AW:TYPECODE=i
-	public double input9;	//AW:TYPECODE=i
-	public double input10;	//AW:TYPECODE=i
-	public double input11;	//AW:TYPECODE=i
-	public double input12;	//AW:TYPECODE=i
-	public double input13;	//AW:TYPECODE=i
-	public double input14;	//AW:TYPECODE=i
-	public double input15;	//AW:TYPECODE=i
-	public double input16;	//AW:TYPECODE=i
-	public double input17;	//AW:TYPECODE=i
-	public double input18;	//AW:TYPECODE=i
-	public double input19;	//AW:TYPECODE=i
-	public double input20;	//AW:TYPECODE=i
-	String _inputNames[] = { "input1", "input2", "input3", "input4", "input5",
-				 "input6", "input7", "input8", "input9", "input10",
-				 "input11", "input12", "input13", "input14", "input15",
-				 "input16", "input17", "input18", "input19", "input20"};
-//AW:INPUTS_END
+	@Input
+	public double input1;
+	@Input
+	public double input2;	
+	@Input
+	public double input3;	
+	@Input
+	public double input4;	
+	@Input
+	public double input5;	
+	@Input
+	public double input6;
+	@Input
+	public double input7;
+	@Input
+	public double input8;
+	@Input
+	public double input9;
+	@Input
+	public double input10;
+	@Input
+	public double input11;
+	@Input
+	public double input12;
+	@Input
+	public double input13;	
+	@Input
+	public double input14;	
+	@Input
+	public double input15;	
+	@Input
+	public double input16;	
+	@Input
+	public double input17;	
+	@Input
+	public double input18;	
+	@Input
+	public double input19;	
+	@Input
+	public double input20;	
 
-//AW:LOCALVARS
 	String alg_ver = "1.0.01";
```

Comment:
Probably should bump this, if it's even used?

Reply:
done